insights/parsers/kpatch_patches.py

```diff
@@ -0,0 +1,40 @@
+"""
+KpatchPatches - report locally stored kpatch patches
+====================================================
+
+This parser creates a list of the module names of locally
+stored kpatch modules. If no modules are installed, the
+list will be empty.
+
+"""
+
+from .. import parser, Parser
+from insights.specs import Specs
+import re
+
+
+@parser(Specs.kpatch_patch_files)
+class KpatchPatches(Parser):
+    """
+    A parser for getting locally stored kpatch-patch modules
+    """
+    __re_cannot_access = re.compile(r"No such file or directory$")
+    __re_archive = re.compile(r"\.xz$")
```

Comment:
TBH, this `regex` check is very simple, the `str_obj.endswith('.xz')` works in better efficiency than this `re` and the `endswith` looks better too. Would you please change it?

Reply:
All of these unnecessary regexes are removed in the latest patch